src/modules/song/types.ts

```diff
@@ -92,6 +92,7 @@ export interface PostSongRequest {
   readonly ipis?: ReadonlyArray<string>;
   readonly releaseDate?: string;
   readonly publicationDate?: string;
+  readonly coverRemixSample?: boolean;
```

Comment:
I think this would benefit to be changed to `isCoverRemixSample` just to be consistent for `boolean` values.

Reply:
This is how it is defined in the BE for post and get, I wanted to keep it consistent throughout so will keep it as is to not get confused on one vs the other.

Reply:
i'll update it